ios/Mattermost/AppDelegate.m

```diff
@@ -164,6 +189,28 @@ - (BOOL)application:(UIApplication *)application continueUserActivity:(NSUserAct
   return extraModules;
 }
 
+-(void)cleanNotificationsFromChannel:(NSString *)channelId {
```

Comment:
Seems like this and the previous change is just reordering. It it intended?

Reply:
yes intended.